<launch>
    <node pkg="sml_world" name="road_network" type="road_network.py" args="/resources/scenarios/KistaFinal True False" />
    <node pkg="sml_world" name="visualization" type="visualization.py" args="False True False" />
    <node pkg="sml_world" name="sml_world_central" type="sml_world_central.py" />
    <node pkg="sml_world" name="bus_network" type="bus_network.py" />

</launch>
